docs/quickstart.md

```diff
@@ -123,14 +124,22 @@ touch Pipfile && pipenv --python 3.8 && pipenv shell
 
 The following installs the entire AEA package which also includes a <a href="../cli-commands">command-line interface (CLI)</a>. (You can skip this step if you used the install script above: <a href="../quickstart#option-2-using-an-automated-install-script">Option 2 </a>.)
 
+First, check which shell you are using:
+``` bash
+echo "$SHELL"
+```
+
+For bash:
 ``` bash
 pip install open-aea[all]
+pip3 install open-aea-cli-ipfs
 ```
 
-If you are using `zsh` rather than `bash` type
+If you are using `zsh` (default shell in modern MacOS installations) rather than `bash` type
 ``` zsh
 pip install 'open-aea[all]'
 pip install 'open-aea-ledger-ethereum'
+pip3 install 'open-aea-cli-ipfs'
```

Comment:
This isn't installing with [all], and we need it for this quickstart.

Reply:
why pip3 versus pip?

Reply:
yes let's keep it consistent